hooks/000-provide-uids-gids.chroot

```diff
@@ -226,6 +228,8 @@ geoclue:x:120:
 gdm:x:121:
 scanner:x:122:
 colord:x:123:
+pipewire:x:113:
```

Comment:
Same for reuse of gid 113 here. That was previously allocated to docker.

Reply:
Done.